specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-02-02/bms.json

```diff
@@ -0,0 +1,473 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-02-02",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/privateEndpointConnections/{privateEndpointConnectionName}": {
```

Comment:
Please include a "list" API for privateEndpointConnections

Reply:
Yes we need to add, since vault properties has the list already. That is the way portal/private link center will call. List was just an auxillary api. Given we had to deliver on tight deadlines, we added min set. We have this as action item and will do it post this release.